README.md

```diff
@@ -17,7 +17,10 @@ guides, and instructions on building from source.
 
 ## Project news
 
-* 2024-05-23: [IREE joins the LF AI & Data Foundation as a sandbox-stage project](https://lfaidata.foundation/blog/2024/05/23/announcing-iree-a-new-initiative-for-machine-learning-deployment/)
+* 2025-04-02:
+[IREE submits SDXL text-to-image MLPerf inference benchmarks using AMD Instinct™ MI325X](https://rocm.blogs.amd.com/artificial-intelligence/mi325x-accelerates-mlperf-inference/README.html#stable-diffusion-xl-sdxl-text-to-image-mlperf-inference-benchmark)
```

Comment:
Open to suggestions about the text here. Maybe

"using IREE" with active voice:

> AMD submits SDXL text-to-image MLPerf inference benchmarks on AMD Instinct™ MI325X using IREE

or passive voice:

> SDXL text-to-image MLPerf inference benchmarks on AMD Instinct™ MI325X submitted using IREE